sycl/source/detail/platform_impl.cpp

```diff
@@ -126,6 +126,9 @@ static std::vector<DevDescT> getAllowListDesc() {
       valuePtr = &decDescs.back().devDriverVer;
       size = &decDescs.back().devDriverVerSize;
       str += sizeof(driverVerStr) - 1;
+    } else {
+      throw sycl::runtime_error("Unrecognized key in device allowlist",
```

Comment:
allowlist -> allow list

Reply:
It's called allowlist in other messages. So, I think it's better to leave as is.